From df478a32d0f8ae591dc9aca22d666717d5b53285 Mon Sep 17 00:00:00 2001
From: Peleg Vigodny <peleg.vigodny@tandemg.com>
Date: Thu, 06 Aug 2020 10:26:21 +0300
Subject: [PATCH] [SW-5220] hls: tools: add pcie link restore

Add pcie link restore tool to the host tools package.

Change-Id: Ic97314f8957415154dc5b4e3183cd5a1b8a33484
Signed-off-by: Peleg Vigodny <peleg.vigodny@tandemg.com>
---

diff --git a/package/openbmc/openbmc.mk b/package/openbmc/openbmc.mk
index 96f378a..9f827ed 100644
--- a/package/openbmc/openbmc.mk
+++ b/package/openbmc/openbmc.mk
@@ -5,7 +5,7 @@
 OPENBMC_IMAGE = $(OPENBMC_IMAGE_DIR)/$(OPENBMC_IMAGE_NAME)
 OBMC_PHOSPHOR_IMAGE = $(OPENBMC_IMAGE_DIR)/$(OBMC_PHOSPHOR_IMAGE_NAME)
 OBMC_PHOSPHOR_FULLCHIP_IMAGE = $(OPENBMC_IMAGE_DIR)/$(OBMC_PHOSPHOR_FULLCHIP_IMAGE_NAME)
-HLS_TOOLS_PACKAGES = frugen-native hl-pcie-sw-update-native
+HLS_TOOLS_PACKAGES = frugen-native hl-pcie-sw-update-native hl-pci-link-restore-native
 VERSION_FILE := $(OPENBMC_DEPLOY_DIR)/localversion-habanalabs
 
 define SET_VERSION_INFO
diff --git a/scripts/make/hls/Makefile b/scripts/make/hls/Makefile
index 7d2d38c..3494a3f 100644
--- a/scripts/make/hls/Makefile
+++ b/scripts/make/hls/Makefile
@@ -6,7 +6,7 @@
 export OBMC_PHOSPHOR_FULLCHIP_IMAGE_NAME = $(OBMC_PHOSPHOR_IMAGE_BASENAME).static.mtd.all.tar
 export OPENBMC_IMAGE_DIR = $(OPENBMC_WORK_DIR)/tmp/deploy/images/hls
 export OPENBMC_DEPLOY_DIR = $(OUTPUT_DIR)/build/openbmc-custom
-export HLS_TOOLS_DIRS = frugen hl-pcie-sw-update
+export HLS_TOOLS_DIRS = frugen hl-pcie-sw-update hl-pci-link-restore
 unexport TEMPLATECONF
 
 ABS_OPENBMC_WORK_DIR = $(OPENBMC_DEPLOY_DIR)/$(OPENBMC_WORK_DIR)
